models/background.py

```diff
@@ -92,32 +95,6 @@ def method(self, value):
         doc="Noise RMS of the background (in units of counts), as a best representative value for the entire image."
     )
 
-    provenance_id = sa.Column(
-        sa.ForeignKey('provenances.id', ondelete="CASCADE", name='backgrounds_provenance_id_fkey'),
-        nullable=False,
-        index=True,
-        doc=(
-            "ID of the provenance of this Background object. "
-            "The provenance will contain a record of the code version"
-            "and the parameters used to produce this Background object."
-        )
-    )
```

Comment:
Marking this for consistency with the others - just checking whether you intended to delete provenance_id. I noticed in some models you only delete the provenance relationship and in others you kill the id too, but I haven't put together if there is reason why or if its a mistake.